README.rst

```diff
@@ -173,21 +173,10 @@ The analysis workflow is simple and consists of two above-mentioned stages:
 There is a serial workflow that does the steps sequentially, but this demo
 represents a better use case for workflow tools capable of parallel execution.
 
-We shall use the `CWL <http://www.commonwl.org/v1.0/>`_ workflow specification
+We shall use the `Snakemake <https://snakemake.readthedocs.io/en/stable/>`_ workflow specification
 to express the computational workflow:
 
-- `workflow definition <workflow/workflow.cwl>`_
-
-and its individual steps:
-
-- `process collision data <workflow/analyse_data.cwl>`_
-- `process simulated data <workflow/analyse_mc.cwl>`_
-- `produce final plot <workflow/make_plot.cwl>`_
-
-Note that we can also use the Snakemake workflow specification (see
-`reana-snakemake.yaml <reana-snakemake.yaml>`_):
-
-- `workflow definition <workflow/snakemake/Snakefile>`_
+- `workflow definition <workflow/Snakefile>`_
```

Comment:
Since everything is now included in Snakefile, and since we remove Serial and CWL, what about showing the full content of Snakefile here, rather than only linking there? OT1H, it might be good for Snakemake users to feel at home. OTOH, it could make README file rather long by some 60 lines. Perhaps make an experiment how it would like and keep one or the other?

Reply:
Showing the whole Snakefile is indeed kind of long, but I like how it looks overall, and it's not very unbalanced!